lib/gds_api/email_alert_api.rb

```diff
@@ -247,6 +247,24 @@ def send_subscription_verification_email(address:, frequency:, topic_id:)
     )
   end
 
+  # Unsubscribe all users for a subscription list
+  # optionally send a notification email explaining the reason
+  #
+  # @param [string]               subscriber_list_id    Identifier for the subscription list
+  # @param [string] (optional)    body                  Optional email body to send to alert users they are being unsubscribed
+  # @param [string] (optional)    sender_message_id     A UUID to prevent multiple emails for the same event
+  #                                                     generated as a v5 UUID comprising content_id + public_updated_at
```

Comment:
I think leave out this line.

email-alert-api doesn't actually care how the UUID is generated, it's a detail which the user of this API needs to care about and this comment currently suggests that this endpoint will *only* work for content ID-based lists, which isn't true.

We talked about v5 UUIDs in the email-alert-api PR because we wanted to make sure there would be a feasible way to come up with a UUID for the events we currently care about (unpublishings which have both a content ID and a public timestamp, but no other useful identifying information), but another approach would work equally well so long as it's consistent (and other subscriber lists could be bulk-unsubscribed so long as the sender_message_id used doesn't conflict).